packages/jest-jasmine2/vendor/jasmine-2.5.2.js

```diff
@@ -1612,7 +1612,8 @@ getJasmineRequireObj().buildExpectationResult = function() {
       matcherName: options.matcherName,
       message: message(),
       stack: stack(),
-      passed: options.passed
+      passed: options.passed,
+      error: options.error
```

Comment:
Anything added to the Jasmine fork should have an appropriate comment, so it's easier to migrate to newer versions later, e.g.:
```javascript
// CUSTOM JEST CHANGE: we pass error message to the result.
error: options.error
```